src/components/select-field/BaseSelectField.js

```diff
@@ -247,22 +281,25 @@ class BaseSelectField extends React.Component<Props, State> {
                 }
                 break;
             default: {
-                stopDefaultEvent(event);
-                const lowerCaseKey = key.toLowerCase();
-                const optionIndex = options.findIndex(
-                    option => option.displayText.toLowerCase().indexOf(lowerCaseKey) === 0,
-                );
+                if (!shouldShowSearchInput) {
+                    stopDefaultEvent(event);
+                    const lowerCaseKey = key.toLowerCase();
+                    const optionIndex = options.findIndex(
```

Comment:
this is not supported in IE11. is there a polyfill? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex

Reply:
I can use findIndex from Lodash, it looks like it supports IE11.